config/initializers/airbrake.rb

```diff
@@ -2,9 +2,9 @@
   errbit_uri = Plek.find_uri("errbit")
 
   Airbrake.configure do |config|
-    config.api_key = ENV["ERRBIT_API_KEY"]
-    config.host = errbit_uri.host
-    config.secure = errbit_uri.scheme == "https"
+    config.project_key = ENV["ERRBIT_API_KEY"]
+    config.project_id = 9999 # This is required by airbrake but ignored by errbit.
+    config.host = errbit_uri.scheme + '://' + errbit_uri.host
```

Comment:
you can use `config.host = errbit_uri.to_s` here rather than building the uri yourself.